packages/compass-indexes/src/components/search-indexes-table/search-indexes-table.tsx

```diff
@@ -215,18 +255,30 @@ export const SearchIndexesTable: React.FunctionComponent<
             onEditIndex={onEditIndex}
             onRunAggregateIndex={(name) => {
               openCollectionWorkspace(namespace, {
-                initialPipeline: getInitialSearchIndexPipeline(name),
                 newTab: true,
+                ...(isVectorSearchIndex
+                  ? {
+                      initialPipelineText:
+                        getInitialVectorSearchIndexPipelineText(name),
+                    }
+                  : {
+                      initialPipeline: getInitialSearchIndexPipeline(name),
+                    }),
```

Comment:
Seems a bit weird that we are passing text for one, but parsed version for another. Can we normalize this?

Reply:
I'll chat with product on what we want to show here, text gives the option to give an invalid pipeline and include code comments. We can do something similar to what we do for regular search indexes.

Reply:
Also, a bit related, started a thread on slack on what we want the pipeline text to look like. Based on that I'll align them. If no one gets back I'll just update them both to text with a comment. Might wrap it into the follow up branch / pr for urls.